alerting/src/test/kotlin/org/opensearch/alerting/resthandler/MonitorRestApiIT.kt

```diff
@@ -956,10 +966,13 @@ class MonitorRestApiIT : AlertingRestTestCase() {
 
     fun `test monitor stats when disabling and re-enabling scheduled jobs with existing monitor`() {
         // Enable Monitor jobs
-        enableScheduledJob()
+
+        var response = enableScheduledJob()
```

Comment:
Could you explain how this is being used in the `waitUntil`s? 

My understanding is that `waitUntil` essentially performs the `booleanSupplier` check periodically. If the check is `true`, or the `maxWaitTime` has been met, it will proceed past the `waitUntil` block; if `false`, it will continue performing the check.

Since we're using the `waitUntil` to force a pause in the test execution in this context, would it make sense to just set the `breakSupplier` to `false` that way the block always waits for the timeout?
E.g.,
```
if (isMultiNode) OpenSearchTestCase.waitUntil({
            return@waitUntil false
        }, 2, TimeUnit.SECONDS)
```
I mostly curious if this is an option because I'm not sure using `enableScheduledJob` in this way will do what we want. The constructor of that function doesn't suggest it returns a nullable `Response`. So I'm not sure that object will ever be `null`. Also, `response` doesn't get set back to `null` after line `975`, so the `response` that's checked by the subsequent `waitUntil`s probably won't be `null` either.

Reply:
I think I had some misunderstanding in enableScheduledJob(). The main purpose of setting a response variable is to know if the PUT request in enableScheduledJob() went through all the process before proceeding to the next line. So, my assumption was that the response will be null until the request is made, which will set it to non-null variable and change `booleanSuppler` so that it can proceed afterwards. I will make a change that you suggested to other tests that I can applied to. Thanks!